tests/wpt/metadata/html/browsers/browsing-the-web/overlapping-navigations-and-traversals/nav-cancelation-2.sub.html.ini

```
@@ -1,3 +1,4 @@
 [nav-cancelation-2.sub.html]
+  expected: TIMEOUT
   [grandparent cancels a pending navigation in a cross-origin grandchild]
-    expected: FAIL
+    expected: TIMEOUT
```

Because we don't get TIMEOUT twice, I think this was an intermittent TIMEOUT.

```suggestion
  [grandparent cancels a pending navigation in a cross-origin grandchild]
    expected: FAIL
```